idd/V8-2-0-Energy+.idd

```diff
@@ -772,7 +772,8 @@ ConvergenceLimits,
        \minimum 2
 
 ProgramControl,
-       \memo used to support various efforts in time reduction for simulation including threading
+       \memo Used to support various efforts in time reduction for simulation including threading
```

Comment:
@rglazer2 This is the wrong IDD file to put the changes inn.  Should be idd/Energy+.idd.in